### PR TITLE
sspBC bid adapter: update to v5.0 

### PR DESCRIPTION
**Pull request by wojciech-bialy-wpm**
## Type of change

- [ ] Bugfix
- [x] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change
- read and send first party data
- send placement size / no of times this size was called (improves site/slot detection)
- send User Agent client hints

### CODE REVIEW

**Comment by robertrmartinez** [comment]
Please add unit tests to cover this PR

**Comment by wojciech-bialy-wpm** [comment]
I've added tests for first party data (pbsize) and placement size (pbslot)

File: modules/sspBCBidAdapter.js [hunk 1]
```diff
@@ -153,22 +156,37 @@ const mapBanner = slot => {
...
+  const { adUnitCode, bidId, params = {}, ortb2Imp = {} } = slot;
+  const { id, siteId } = params;
+  const { ext = {} } = ortb2Imp;
+
+  /*
+     check max size for this imp, and check/store number this size was called (for current view)
+     send this info as ext.pbsize
+  */
+  const slotSize = slot.sizes.length ? slot.sizes.reduce((prev, next) => prev[0] * prev[1] <= next[0] * next[1] ? next : prev).join('x') : '1x1';
+  adSizesCalled[slotSize] = adSizesCalled[slotSize] ? adSizesCalled[slotSize] += 1 : 1;
```
Comment (msm0504) [inline modules/sspBCBidAdapter.js:168]: This seems to be assigning the value twice. `+= 1` can be replaced with `+ 1`

**Review by msm0504 — CHANGES_REQUESTED** [review]

File: modules/sspBCBidAdapter.js (hunk 1 above)
Comment (wojciech-bialy-wpm) [reply modules/sspBCBidAdapter.js:168]: Right! Fixing it now.

**Review by msm0504 — APPROVED** [review]

**Pull request merged**